src/olympia/templates/copyright.html

```diff
@@ -1,13 +1,13 @@
 <div id="copyright">
   <p id="footer-links">
     {% block footer_links %}
-      <a href="http://www.mozilla.org/privacy/websites/">
+      <a href="https://www.mozilla.org/en-US/privacy/websites/">
         {{ _('Privacy Policy') }}
       </a> &nbsp;|&nbsp;
-      <a href="http://www.mozilla.org/about/legal.html">
+      <a href="https://www.mozilla.org/en-US/about/legal/">
         {{ _('Legal Notices') }}
       </a> &nbsp;|&nbsp;
-      <a href="http://www.mozilla.org/legal/fraud-report/index.html">
+      <a href="https://www.mozilla.org/en-US/about/legal/fraud-report/">
```

Comment:
Looks like the there are no other translations e.g. whatever locale is used in place of en-US you always end up at en-US. If that's the case leaving the en-US is fine since it does save the redirect.